vpr/src/route/rr_graph.cpp

```diff
@@ -741,11 +744,25 @@ void create_rr_graph(const t_graph_type graph_type,
     }
 
     if (is_flat) {
+        short delayless_switch = OPEN;
+        if (load_rr_graph) {
+            const auto& rr_switches = device_ctx.rr_graph.rr_switch();
+            for (int switch_id = 0; switch_id < static_cast<int>(rr_switches.size()); switch_id++){
```

Comment:
Small nit-pick; instead of static casting the bound to an int, you can make switch_id a `size_t`. There is no usage of switch_id that requires it to be an int as far as I can tell.